pages/api/id/[form]/owners.ts

```diff
@@ -38,5 +40,40 @@ export async function getEmailListByFormID(
   //Could not find formID in the path
   return res.status(400).json({ error: "Malformed API Request FormID not define" });
 }
+/**
+ * It will activate and deactivate all the owners associated to a specific form.
+ * This function is expecting to find a payload in the request body like:
+ * { "email": "forms@cds.ca", "active":"0/1"}
+ * "email": The email of the user associated to the form
+ * "active": Boolean value indicating whether the form owner is active or not
+ * @param req
+ * @param res
+ */
+export async function activateOrDeactivateFormOwners(
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> {
+  //Extracting req body
+  const requestBody =
+    req.body && Object.keys(req.body).length > 0 ? JSON.parse(req.body) : undefined;
```

Comment:
Think checking if req.body is truthy is enough of a check. No need for Object.keys